tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -450,28 +451,47 @@ public function makeDavRequest(
 	 * @param string|null $height
 	 *
 	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
 	 */
 	public function downloadPreviews(string $user, ?string $path, ?string $doDavRequestAsUser, ?string $width, ?string $height):void {
-		$user = $this->getActualUsername($user);
-		$doDavRequestAsUser = $this->getActualUsername($doDavRequestAsUser);
 		$urlParameter = [
 			'x' => $width,
 			'y' => $height,
 			'forceIcon' => '0',
 			'preview' => '1'
 		];
-		$this->response = $this->makeDavRequest(
+		$davVersion = $this->getDavPathVersion();
+		if ($davVersion === WebDavHelper::DAV_VERSION_SPACES) {
+			$fileId = urlencode(strstr($this->getFileIdForPath($doDavRequestAsUser ?? $user, $path), '!', true));
+
+			$eTag = str_replace('"', '', $this->getFileEtagForPath($doDavRequestAsUser ?? $user, $path));
+			$url = $this->getBaseUrl() . '/remote.php/dav/spaces/' . $fileId . '/';
```

Comment:
`$url` can be used like this
```suggestion
                $url = $this->getBaseUrl() . '/remote.php';
		$davVersion = $this->getDavPathVersion();
		if ($davVersion === WebDavHelper::DAV_VERSION_SPACES) {
			$fileId = urlencode(strstr($this->getFileIdForPath($doDavRequestAsUser ?? $user, $path), '!', true));

			$eTag = str_replace('"', '', $this->getFileEtagForPath($doDavRequestAsUser ?? $user, $path));
			$url .= '/dav/spaces/' . $fileId . '/';
```